tools/bigquery-to-xml/test/results.txt

```diff
@@ -0,0 +1,475 @@
+<?xml version="1.0" ?>
+<results>
+	<row>
+		<repository>
+			<url>https://github.com/zzzsochi/Flask-Gravatar</url>
```

Comment:
nit: Will this table get updated if these repos are deleted? If so that will break your test :)
IMO there's no reason to make your test dependent on zzzsochi not deleting their fork of FlaskGravatar.
I also don't like the idea of having random GH user ids checked in. Instead lets use contributors to this repo!

I'd suggest just querying this repo something like:
```sql
SELECT
  *
FROM
  `bigquery-public-data.github_repos.commits`
WHERE
  "GoogleCloudPlatform/professional-services" IN UNNEST(repo_name)
ORDER BY
  committer.time_sec ASC
LIMIT
  5
```

Reply:
This BQ table hasn't been updated since it was created in 2016 - looks like it's fairly static.

Fair point on the GitHub ID check-ins. Since the table hasn't been updated in so long I don't think it has any commits for this repo - for now I have it pointing to JetBrains, but there are still some IDs attached to it ofc. 

`SELECT * 
FROM `bigquery-public-data.samples.github_nested` 
WHERE repository.url LIKE "%/JetBrains/kotlin%" 
ORDER BY repository.created_at DESC 
LIMIT 5`

If we want to avoid the problem overall, happy to point to a different public dataset as well!